src/test/java/org/eclipse/pass/schema/SchemaServletTest.java

```diff
@@ -0,0 +1,38 @@
+package org.eclipse.pass.schema;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.io.StringReader;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+class SchemaServletTest {
```

Comment:
This needs to test the HTTP methods. Testing utility methods is ok. (Those utility methods should not be public though.)

Reply:
Unit test for doPost() added in commit "add test for full service (doPost)"